activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -10,7 +10,7 @@
 require "active_record/connection_adapters/sqlite3/schema_dumper"
 require "active_record/connection_adapters/sqlite3/schema_statements"
 
-gem "sqlite3", "~> 1.3", ">= 1.3.6"
+gem "sqlite3", "~> 1.4"
```

Comment:
We might need to add changelog entry to note that Active Record is compatible with `sqlite3` `~> 1.4` since Rails 6.0.